packages/server/src/express/middleware.ts

```diff
@@ -86,7 +89,8 @@ const factory = (options: MiddlewareOptions): Handler => {
                 zodSchemas,
                 logger: options.logger,
             });
-            if (manageCustomResponse) {
+            if (sendResponse === false) {
```

Comment:
I think here we have a problem... `sendReponse` is `undefined`... I think this will be `true` always

Reply:
When not set `sendResponse` will be undefined, so `=== false` evaluates to `false`. I think it works. @chemitaxis